web/modules/custom/dbcdk_community/src/Content/FlaggableContentRepository.php

```diff
@@ -182,4 +204,47 @@ public function getContentByIdAllFlags($flag_id) {
     return $flagged_content;
   }
 
+  /**
+   * Get collections for a FlaggableContent object.
+   *
+   * This method provides a simple way to fetch collections for a
+   * FlaggableContent object.
+   *
+   * @param \Drupal\dbcdk_community\Content\FlaggableContent $content
+   *   The FlaggableContent object we wish to fetch collections from.
+   * @param string $type
+   *   The type of collection we wish to fetch.
+   *
+   * @return \DBCDK\CommunityServices\Model\ImageCollection[]|\DBCDK\CommunityServices\Model\VideoCollection[]
+   *   On array of collections for the provided FlaggableContent object.
+   */
+  public function getCollections(FlaggableContent $content, $type = 'image') {
```

Comment:
This would also make the separation of the test more logical. I'll do that right away.